cli/command/stack/kubernetes/cli.go

```diff
@@ -23,20 +23,28 @@ type Options struct {
 }
 
 // NewOptions returns an Options initialized with command line flags
-func NewOptions(flags *flag.FlagSet) Options {
+func NewOptions(flags *flag.FlagSet, namespace ...string) Options {
 	var opts Options
-	if namespace, err := flags.GetString("namespace"); err == nil {
-		opts.Namespace = namespace
+	if len(namespace) > 0 {
+		opts.Namespace = namespace[0]
+	} else if nm, err := flags.GetString("namespace"); err == nil {
```

Comment:
in what situation would `flags.GetString("namespace")` give anything different than `namespace[0]`? If we only expect a single namespace to be passed, it should not take more than one